deploy/roles/cometbft/templates/devnet/config/genesis.json

```diff
@@ -308,8 +308,9 @@
                   "lp_denom": "dex/pool/dango/usdc",
                   "pool_type": {
                     "xyk": {
-                      "order_spacing": "1",
-                      "reserve_ratio": "0"
+                      "limit": 30,
+                      "reserve_ratio": "0",
+                      "spacing": "1"
```

Comment:
doesn't really matter until we have some kind of trading bot live but it could be a good idea to set the spacing to appropriate values based on the assets pair

or maybe 1 is good enough for all tokens?